### PR TITLE
[IMP] web: remove listview tooltip for text fields

### PR DESCRIPTION
**Pull request by bastvdn**
Description of the issue/feature this PR addresses:

we figured out that the tooltip is useless for text fields because the entirety of the value is displayed anyway, so we just decided to remove tooltips in listview for text fields

Small note about the tests/tours removal:

IMO we need to remove those test code snipets.
the problem is that the html element td[name='name'] does not contain information about the text inside the description while the focus
is still on the record. The only way we had to know what was inside the textarea before unfocusing was through the datatooltip
which doesn't exist anymore.

Linked enterprise PR: https://github.com/odoo/enterprise/pull/51501

Task-3186593

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/test_main_flows/static/tests/tours/main_flow.js [hunk 1]
```diff
@@ -556,15 +560,15 @@ stepUtils.autoExpandMoreButtons('.o_form_saved'),
     extra_trigger: ".modal:not(.o_inactive_modal) .modal-title:contains('Order Lines')",
     content: _t('Save & New'),
     position: 'right',
-}, {
+}, 
+{
```
Comment (xavierbol) [inline odoo/addons/test_main_flows/static/tests/tours/main_flow.js:564]: Unnecessary diff 😇

**Review by xavierbol — COMMENTED** [review]
Thanks for your work, I left some comments. 🙂

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for your work. 🙂 

I will let @odoo/rd-framework-js doing the final review. 🙂

**Comment by aab-odoo** [comment]
Hello @bastvdn A qunit test would be nice for the text field tooltip thing. See test "boolean field has no title (data-tooltip)" for inspiration.

**Comment by bastvdn** [comment]
Hello @aab-odoo 
Thanks for the review :)
Is the test i wrote enough? Basically the same as for the boolean

File: addons/web/static/tests/views/list_view_tests.js [hunk 2]
```diff
@@ -1732,6 +1732,16 @@ QUnit.module("Views", (hooks) => {
         assert.strictEqual(target.querySelector(".o_data_cell").getAttribute("data-tooltip"), null);
     });
 
+    QUnit.only("text field has no title (data-tooltip)", async function (assert) {
```
Comment (aab-odoo) [inline addons/web/static/tests/views/list_view_tests.js:1735]: QUnit.test

Also, you can squash this commit with the first one.

Besides thats, test is ok :-)

Comment (bastvdn) [reply addons/web/static/tests/views/list_view_tests.js:1735]: yes obviously ^^'
thanks for the review

**Review by aab-odoo — APPROVED** [review]

**Comment by xavierbol** [comment]
robodoo rebase-ff r+

**Comment by aab-odoo** [comment]
robodoo retry

**Comment by xavierbol** [comment]
robodoo r- 

it seems the staging crashes because of those changes. 😬

**Comment by xavierbol** [comment]
https://runbot.odoo.com/runbot/build/54150788
```txt
odoo.addons.test_main_flows.tests.test_flow:75FAIL: TestUi.test_01_main_flow_tour Traceback (most recent call last):   File "/data/build/odoo/odoo/addons/test_main_flows/tests/test_flow.py", line 75, in test_01_main_flow_tour     self.main_flow_tour()   File "/data/build/odoo/odoo/addons/test_main_flows/tests/test_flow.py", line 69, in main_flow_tour     self.start_tour("/web", 'main_flow_tour', login="admin", timeout=180)   File "/data/build/odoo/odoo/tests/common.py", line 1808, in start_tour     return self.browser_js(url_path=url_path, code=code, ready=ready, **kwargs)   File "/data/build/odoo/odoo/tests/common.py", line 1787, in browser_js     self.fail('%s\n\n%s' % (message, error)) AssertionError: The test code "odoo.startTour('main_flow_tour', {"stepDelay": 0, "keepWatchBrowser": false, "startUrl": "/web"})" failed  Tour main_flow_tour failed at step Enter an email address (trigger: .o_field_widget[name=email] input)
--


[odoo.addons.test_main_flows.tests.test_flow:75](https://github.com/odoo/odoo/blob/50763232e943af65b00982871ca429b5c9ba26f2/odoo/addons/test_main_flows/tests/test_flow.py#L75)
FAIL: TestUi.test_01_main_flow_tour
Traceback (most recent call last):
  File "/data/build/odoo/odoo/addons/test_main_flows/tests/test_flow.py", line 75, in test_01_main_flow_tour
    self.main_flow_tour()
  File "/data/build/odoo/odoo/addons/test_main_flows/tests/test_flow.py", line 69, in main_flow_tour
    self.start_tour("/web", 'main_flow_tour', login="admin", timeout=180)
  File "/data/build/odoo/odoo/tests/common.py", line 1808, in start_tour
    return self.browser_js(url_path=url_path, code=code, ready=ready, **kwargs)
  File "/data/build/odoo/odoo/tests/common.py", line 1787, in browser_js
    self.fail('%s\n\n%s' % (message, error))
AssertionError: The test code "odoo.startTour('main_flow_tour', {"stepDelay": 0, "keepWatchBrowser": false, "startUrl": "/web"})" failed

Tour main_flow_tour failed at step Enter an email address (trigger: .o_field_widget[name=email] input)
```

**Comment by bastvdn** [comment]
@xavierbol 
I think we do need to keep this line to remove the exit the cell focus or the customer edit dialogbox won't display
image

**Comment by xavierbol** [comment]
> @xavierbol I think we do need to keep this line to remove the exit the cell focus or the customer edit dialogbox won't display image

Oki thanks for the update. 🙂

**Review by xavierbol — APPROVED** [review]
robodoo r+

**Comment by xavierbol** [comment]
robodoo r+

**Comment by xavierbol** [comment]
robodoo r+
